File: roteiro_03/java_logic/desafios/desafio3.java
package roteiro_03.java_logic.desafios;

import java.util.Scanner;

public class desafio3 {
    public static void main (String[] args){
        Scanner sc = new Scanner(System.in);
        double soma = 0;

        for (int i = 1; i < 5; i++) {
            System.out.print("Digite sua nota:"+ i + ":");
            double nota = sc.nextDouble();
            soma +=nota; 
        }
        double media = soma/5;
        System.out.println("A média final das notas é:"+ media);
        sc.close();
    }

    
}

File: roteiro_03/cpp_logic/desafio3.cpp
#include <iostream>
using namespace std;

int main() {
    float nota;
    float soma = 0; 
    float media;

    cout << "--- Calculadora de Media ---" << endl;


    for (int i = 1; i <= 5; i++) {
        cout << "Digite a nota " << i << ": ";
        cin >> nota;
        soma = soma + nota; 
    }


    media = soma / 5;

    cout << "\nA soma de todas as notas e: " << soma << endl;
    cout << "A media final e: " << media << endl;

    return 0;
}